CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 All notable changes to the code will be documented in this file.
 
+## 2.0.1
+
+Bug Fixes
```

Comment:
This is not a bug fix, but an enhancement to the API.

Reply:
@johnpapa Is 2.0.1 okay for enhancement or should I bump minor version for this change?

Reply:
Updated to 2.1.0